examples/example-myst/myst.yml

```diff
@@ -17,18 +17,18 @@ project:
   bibliography: []
   exports:
     - format: typst
-      template: typst-thesis.typ
+      template: ../../thesis-template
       output: Exports/MyST-Thesis.pdf
       # toc: _toc.yml
       articles:
-      - file: examples/example-myst/00-Front-Matter/01-Declaration.md
-      - file: examples/example-myst/00-Front-Matter/02-Abstract.md
-      - file: examples/example-myst/00-Front-Matter/03-Preface.md
-      - file: examples/example-myst/00-Front-Matter/04-Acknowledgements.md
-      - file: examples/example-myst/00-Front-Matter/05-Dedication.md
-      - file: examples/example-myst/00-Front-Matter/06-Glossary.md
-      - file: examples/example-myst/01-Chapters/01-Introduction.md
-      - file: examples/example-myst/01-Chapters/02-Chapter.md
+        - file: 00-Front-Matter/01-Declaration.md
```

Comment:
The myst project should be distinct, eventually you might spin the `template` folder off into a small stand-alone repo that can be shared and pointed to independently.